docs/writing-policies/raw-policies.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_label: "Raw policies"
+title: "Raw policies"
+---
+
+# Raw policies
+
+From `v1.9.0` onwards, Kubewarden supports the ability to write and execute policies
+outside a Kubernetes cluster, as a generic policy evaluation engine.
+The Policy Server exposes the `/validate_raw` endpoint that can be used to validate
+arbitrary JSON documents against a Kubewarden policy.
+
+## Running the policy server outside Kubernetes
```

Comment:
I think this page should be renamed "run policy-server outside of Kubernetes", and be kept in this "tutorial" section. But I don't think it should include additional info on raw policies.